doc/releases/releases.yml

```diff
@@ -12,6 +12,11 @@
 # If a version might represent an actual number (e.g. 0.80) quote it.
 #
 releases:
+  reef:
+    target_eol: 2025-08-01
```

Comment:
Assuming we intend to get back to the regular release cycle, I would have gone with `2025-06-01` here.

Reply:
I'll adjust this and I'll also address your complaint about the timeline. Ill do this in a PR tomorrow (or possibly as soon as thirty minutes from now, when we're in the CLT).